packages/react-next/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -17,135 +18,124 @@ const LARGE_IMAGE_SIZE = 71;
+    imageSize,
+    focused,
+  });
+
+  const { className, ...nativeProps } = getNativeProps<{ className: string }>(rest, inputProperties);
+
+  const onRenderLabel = (): JSX.Element => {
     return (
-      <div className={this._classNames.root}>
-        <div className={this._classNames.choiceFieldWrapper}>
-          <input
-            aria-label={ariaLabel}
-            id={id}
-            className={css(this._classNames.input, className)}
-            type="radio"
-            name={name}
-            disabled={disabled}
-            checked={checked}
-            required={required}
-            {...nativeProps}
-            onChange={this._onChange}
-            onFocus={this._onFocus}
-            onBlur={this._onBlur}
-          />
-          {onRenderField(this.props, this._onRenderField)}
-        </div>
-      </div>
+      <span id={props.labelId} className="ms-ChoiceFieldLabel">
+        {props.text}
+      </span>
     );
-  }
-
-  private _onChange = (evt: React.FormEvent<HTMLInputElement>): void => {
-    const { onChange } = this.props;
-    if (onChange) {
-      onChange(evt, this.props);
-    }
-  };
-
-  private _onBlur = (evt: React.FocusEvent<HTMLElement>) => {
-    const { onBlur } = this.props;
-    if (onBlur) {
-      onBlur(evt, this.props);
-    }
-  };
-
-  private _onFocus = (evt: React.FocusEvent<HTMLElement>) => {
-    const { onFocus } = this.props;
-    if (onFocus) {
-      onFocus(evt, this.props);
-    }
   };
 
-  private _onRenderField = (props: IChoiceGroupOptionProps): JSX.Element => {
-    const { id, imageSrc, imageAlt = '', selectedImageSrc, iconProps } = props;
+  const renderField = (): JSX.Element => {
+    const { imageAlt = '', selectedImageSrc } = props;
 
-    const imageSize = props.imageSize ? props.imageSize : { width: 32, height: 32 };
+    const currentImageSize = props.imageSize ? props.imageSize : { width: 32, height: 32 };
 
-    const onRenderLabel = props.onRenderLabel
-      ? composeRenderFunction(props.onRenderLabel, this._onRenderLabel)
-      : this._onRenderLabel;
+    const renderedLabel = props.onRenderLabel
+      ? composeRenderFunction(props.onRenderLabel, onRenderLabel)
+      : onRenderLabel;
 
-    const label = onRenderLabel(props);
+    const label = renderedLabel(props);
 
     return (
-      <label htmlFor={id} className={this._classNames.field}>
+      <label htmlFor={id} className={classNames.field}>
         {imageSrc && (
-          <div className={this._classNames.innerField}>
-            <div className={this._classNames.imageWrapper}>
-              <Image src={imageSrc} alt={imageAlt} width={imageSize.width} height={imageSize.height} />
+          <div className={classNames.innerField}>
+            <div className={classNames.imageWrapper}>
+              <Image src={imageSrc} alt={imageAlt} width={currentImageSize.width} height={currentImageSize.height} />
```

Comment:
I think you can just do this:
```suggestion
              <Image src={imageSrc} alt={imageAlt} width={currentImageSize.width} {...imageSize} />
```